metrics/network/remote_network/README.md

```diff
@@ -34,3 +38,20 @@ $ cd metrics/network/remote_network
 $ bash remote-networking-iperf3.sh -b -i eno1 -u tester -a 10.xxx.xxx.xxx
 
 ```
+
+## Running the `remote-networking-memory-smem.sh` test
+
+`remote-networking-memory-smem.sh` should run in host A. The script needs the following inputs to run:
+- Arguments of the test to run. In this case the argument is `-r` for RSS memory. All arguments should be
+in order.
+- Interface name where swarm will run.
+- User of the host B.
+- IP address of the host B.
+
+The `remote-networking-memory-smem.sh` test may be run manually:
```

Comment:
Use the following commands to run the `remote-networking-memory-smem.sh` test manually: